src/gurt/telemetry.c

```diff
@@ -740,12 +740,16 @@ d_tm_print_node(uint64_t *shmem_root, struct d_tm_node_t *node, int level,
 		show_meta = false;
 
 	if (show_meta) {
-		d_tm_get_metadata(&shortDesc, &longDesc, shmem_root, node,
+		d_tm_get_metadata(&short_desc, &long_desc, shmem_root, node,
 				  NULL);
-		d_tm_print_metadata(shortDesc, longDesc, !stats_printed, format,
-				    stream);
-		D_FREE_PTR(shortDesc);
-		D_FREE_PTR(longDesc);
+		if (format == D_TM_CSV) {
+			if (!stats_printed &&
+			    ((short_desc != NULL) || (long_desc != NULL)))
+				fprintf(stream, ",,,,,");
```

Comment:
maybe comment to describe why that particular number of comma separators

Reply:
done